Foundation/FileManager+Win32.swift

```diff
@@ -497,8 +502,7 @@ extension FileManager {
     }
 
     internal func _isExecutableFile(atPath path: String) -> Bool {
-        // FIXME(compnerd) is there some test that we can perform here?
-        return true
+        return _nonDirectoryFileExists(atPath: path) && _isReadableFile(atPath: path)
```

Comment:
Is there a reason to create the `_nonDirectoryFileExists`?  I think that we can just inline it (unless you have another use that you have planned?).

Reply:
No, mainly for potential future usage and better readability. If you prefer it, I could inline it.

Reply:
Yeah, I think that the inline is going to be better for now.  We can extract it when needed.

Reply:
Done!